### PR TITLE
pbs_snapshot --obfuscate doesn't obfuscate filenames

### PR DESCRIPTION
**Pull request by agrawalravi90**
#### Describe Bug or Feature

Post https://github.com/PBSPro/pbspro/pull/1096 we are still putting the un-obfuscated job ID in the printjob files (e.g., ./mom_priv/jobs/19.osuse15.JB_printjob):

pbs_snapshot --obfuscate -o /tmp

osuse15:/tmp/snapshot_20190515_14_19_30 # ls mom_priv/jobs/
19.osuse15.JB_printjob 20.osuse15.JB_printjob

#### Describe Your Change

Added check for the presence of sensitive values in filenames, if present they'll get replaced with obfuscated values.

#### Attach Test and Valgrind Logs/Output

[before_fix.log](https://github.com/PBSPro/pbspro/files/3188196/before_fix.log)
[after_fix.log](https://github.com/PBSPro/pbspro/files/3188197/after_fix.log)

### CODE REVIEW

File: test/tests/functional/pbs_snapshot_unittest.py [hunk 1]
```diff
@@ -899,6 +899,14 @@ def test_snapshot_obf_stress(self):
                 self.assertIn(ret["out"], ["", None, []], str(val) +
                               " was not obfuscated. Real values:\n" +
                               str(real_values))
+                # Also make sure that no filenames contain the sensitive val
+                cmd = ["find", snap_dir, "-name",  "\'*" + str(val) + "*\'"]
+                ret = self.du.run_cmd(cmd=cmd, as_script=True,
```
Comment (bhroam) [inline test/tests/functional/pbs_snapshot_unittest.py]: You shouldn't need the as_script.  All the arguments are part of find.  It isn't a big deal though.  It's just a little slower.

**Comment by agrawalravi90** [comment]
@bhroam Thanks for the comment, I've removed as_script, it ran fine without it, please let me know if it looks okay.

**Comment by agrawalravi90** [comment]
Thanks @bhroam , @tpai26 or @arungrover can one of you please review this as well? Thanks in advance

**Comment by agrawalravi90** [comment]
@tpai26 and @arungrover can one of you please look at this? it's pretty small

**Pull request merged**